palindrome_finder.py

```
@@ -6,14 +6,19 @@ def is_palindrome(word):
     return word == word[::-1]
 
 
+def clean_word(word):
```

Ruff (0.8.2)

Missing return type annotation for public function `clean_word`

(ANN201)